test_unstructured_ingest/test-ingest-confluence2.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
maybe we name these test files with a suffix other than `1` and `2`. something like `-diff` and `-large`?